packages/OmegaPrint-Core.package/OPPrinter.class/instance/FinalStatement.with.with.with..st

```diff
@@ -0,0 +1,9 @@
+actions
+FinalStatement: aNode with: aTerminal with: aStatement with: anotherTerminal
```

Comment:
should be changed to with:and:and: or be consistent in every method with:with:with:

Reply:
I think I prefer reading with:and:and:

Reply:
We changed it to with:and:and: